addons/binding/org.openhab.binding.logreader/ESH-INF/thing/channels.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="logreader" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<channel-type id="lastErrorEvent">
+		<item-type>String</item-type>
+		<label>Last error event</label>
+		<description>Displays contents of last [ERROR] event</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="lastWarningEvent">
+		<item-type>String</item-type>
+		<label>Last warning event</label>
+		<description>Displays contents of last [WARN] event</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="lastCustomEvent">
```

Comment:
I'd suggest to mark the custom event channels and the log rotated channel as "advanced".